src/js/rtc_session.js

```diff
@@ -885,12 +885,14 @@ export default class RtcSession {
     hangup() {
         this._state.hangup();
     }
+
     /**
-     * Get a promise of MediaRtpStats object for remote audio (from Amazon Connect to client).
+     * Get a promise of two MediaRtpStats objects for remote and user audio (from Amazon Connect to client).
      * @return Rejected promise if failed to get MediaRtpStats. The promise is never resolved with null value.
      */
-    getRemoteAudioStats() {
+    getAudioStats() {
         var timestamp = new Date();
+        var self = this;
         if (this._pc && this._pc.signalingState === 'stable' && this._remoteAudioStream) {
             var audioTracks = this._remoteAudioStream.getAudioTracks();
             return this._pc.getStats(audioTracks[0]).then(function(stats){
```

Comment:
is it possible to get all stats included in this new API (audio, video, all tracks)?